### PR TITLE
Asisov - homework

### PR DESCRIPTION
**Pull request by anek77713**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true
+  },
+  'extends': [
+    'plugin:vue/recommended',
+    'eslint:recommended'
```
Comment (gregurco) [inline .eslintrc.js]: what about standard js style guide?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "www",
```
Comment (gregurco) [inline package.json]: please rename the project

**Review by gregurco — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (anek77713) [reply .eslintrc.js]: done

File: package.json (hunk 2 above)
Comment (anek77713) [reply package.json]: done

**Pull request merged**